Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnectSettingsSO.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using AnkrSDK.WalletConnectSharp.Core.Models;
+using AnkrSDK.WalletConnectSharp.Unity.Models.DeepLink;
+using UnityEngine;
+
+namespace AnkrSDK.WalletConnectSharp.Unity
+{
+	[CreateAssetMenu(fileName = "WalletConnectSettings", menuName = "AnkrSDK/WalletConnect/Settings")]
+	public class WalletConnectSettingsSO : ScriptableObject
+	{
+		[SerializeField] private Wallets _defaultWallet = Wallets.MetaMask;
+		[SerializeField] private bool _autoSaveAndResume = true;
+		[SerializeField] private bool _createNewSessionOnSessionDisconnect = true;
+		[SerializeField] private int _connectSessionRetryCount = 3;
+		[SerializeField] private string _customBridgeUrl = "https://testbridge.yartu.io/";
```

Comment:
change default bridge to a working one(and we will consider to get a stable bridge later)